### ISSUE TITLE
SDK + USB stream

### ISSUE THREAD

**Issue by alezana** [idsc-frazzoli/boson_camera#5]
SDK should configure the camera:
- [ ] Flat field of view (FFC) to automatic
- [ ] Camera output to 16-bit
- [ ] agc setting (first check that they actually affect the video stream captured with v4l2)
But video stream is available only via BosonUSB

**Comment by andy2804** [comment]
Isn't FFC already automatic? (The clicking noise you hear from the shutter)
Camera output is 16bit

**Comment by alezana** [comment]
yes but I wanna check if the setting of the camera through sdk change the raw stream of data via usb
